code/SubsitesVirtualPage.php

```diff
@@ -207,4 +207,13 @@ public function init(){
 		
 		Subsite::$disable_subsite_filter = $origDisableSubsiteFilter;
 	}
+	
+	public function allowedActions ($limitToClass = NULL) {
```

Comment:
It doesn't need to be, it's matching the method signature of the one it's overriding.